tests/foreman/api/test_hostcollection.py

```diff
@@ -379,21 +419,64 @@ def test_negative_create_with_invalid_name(self):
                     entities.HostCollection(name=name, organization=self.org).create()
 
     @tier1
-    @stubbed()
     def test_positive_add_subscription(self):
-        """Try to add a subscription to a host collection
+        """Try to bulk add a subscription to members of a host collection.
 
         :id: c4ec5727-eb25-452e-a91f-87cafb16666b
 
         :steps:
 
             1. Create a new or use an existing subscription
-            2. Add the subscription to the host collection
+            2. Add the subscription to the members of the host collection
 
-        :expectedresults: The subscription was added to the host collection
+        :expectedresults: The subscription was added to hosts in the host collection
 
         :CaseImportance: Critical
         """
+        # Create an activate key for this test
+        act_key = entities.ActivationKey(
+            content_view=self.content_view, environment=self.lce, organization=self.org
+        ).create()
+        # this command creates a host collection and "appends", makes available, to the AK
+        act_key.host_collection.append(entities.HostCollection(organization=self.org).create())
+        # Move HC from Add tab to List tab on AK view
+        act_key = act_key.update(['host_collection'])
+        # Create a product to be bulk added to members of Host Collection
+        product = entities.Product(organization=self.org).create()
+        prod_name = product.name
+        product_subscription = entities.Subscription().search(
+            query={'search': f'name={prod_name}'}
+        )[0]
+        with VirtualMachine(distro=DISTRO_RHEL7) as client1, VirtualMachine(
+            distro=DISTRO_RHEL7
+        ) as client2:
+            clients = [client1, client2]
+            for client in clients:
```

Comment:
Since you never refer to `clients` again, I would just:
```suggestion
            for client in [client1, client2]:
```

Reply:
I see, OK